symphony/app/fbcnms-projects/magmalte/app/components/JsonEditor.js

```diff
@@ -121,23 +115,25 @@ export default function JsonEditor<T>(props: Props<T>) {
           </Grid>
         </Grid>
 
-        <Paper className={classes.configBody}>
-          <Grid container item spacing={3}>
-            <Grid item xs={12}>
-              {error !== '' && <FormLabel error>{error}</FormLabel>}
-            </Grid>
-            <Grid item xs={12}>
-              <ReactJson
-                src={content}
-                enableClipboard={false}
-                displayDataTypes={false}
-                onAdd={handleChange}
-                onEdit={handleChange}
-                onDelete={handleChange}
-              />
-            </Grid>
-          </Grid>
-        </Paper>
+        <Grid
+          container
+          className={classes.configBody}
+          alignItems="stretch"
+          item
+          xs={12}>
+          {error !== '' && <FormLabel error>{error}</FormLabel>}
+          <textarea
```

Comment:
this change seems to be removing my latest change, can you resolve this by using ReactJson component instead of textarea. Thanks

Reply:
Sorry about that, fixed